samples/heroku/main.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from slack_bolt import App
+
+logging.basicConfig(level=logging.DEBUG)
+app = App()
+
+
+@app.command("/hello-bolt-python-heroku")
+def hello(payload, ack):
+    user_id = payload["user_id"]
+    ack(f"Hi <@{user_id}>!")
+
+
+from flask import Flask, request
+from slack_bolt.adapter.flask import SlackRequestHandler
+
+flask_app = Flask(__name__)
+handler = SlackRequestHandler(app)
+
+
+@flask_app.route("/slack/events", methods=["POST"])
+def slack_events():
+    return handler.handle(request)
+
+# heroku login
+# heroku create
+# git remote add heroku https://git.heroku.com/xxx.git
+
+# export $SLACK_BOT_TOKEN=xxx
+# export SLACK_SIGNING_SECRET=xxx
+# heroku config:set SLACK_BOT_TOKEN=$SLACK_BOT_TOKEN
+# heroku config:set SLACK_SIGNING_SECRET=$SLACK_SIGNING_SECRET
+# git push heroku main
```

Comment:
nit: do we want to say `main` or `master` here? Git uses `master` by default so `main` may confuse some developer. However, most Heroku users should be familiar with git and know the branch that they want to push.